packages/charts/test/private-api.test.js

```diff
@@ -1,38 +1,43 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, oneEvent } from '@vaadin/testing-helpers';
 import '../vaadin-chart.js';
+import { inflateFunctions } from '../src/helpers.js';
 
 describe('vaadin-chart private API', () => {
   describe('__inflateFunctions', () => {
-    let chart;
-
     beforeEach(() => {
-      chart = fixtureSync('<vaadin-chart></vaadin-chart>');
+      fixtureSync('<vaadin-chart></vaadin-chart>');
     });
 
     it('should inflate whole function strings', () => {
       // eslint-disable-next-line camelcase
       const config = { tooltip: { _fn_formatter: 'function() {return "awesome chart"}' } };
-      chart.__inflateFunctions(config);
+      inflateFunctions(config);
       expect(config.tooltip.formatter).to.be.a('function');
       expect(config.tooltip).to.not.have.property('_fn_formatter');
     });
 
     it('should inflate function body strings', () => {
       // eslint-disable-next-line camelcase
       const config = { tooltip: { _fn_formatter: 'return "awesome chart"' } };
-      chart.__inflateFunctions(config);
+      inflateFunctions(config);
       expect(config.tooltip.formatter).to.be.a('function');
       expect(config.tooltip).to.not.have.property('_fn_formatter');
     });
 
     it('should inflate empty function strings', () => {
       // eslint-disable-next-line camelcase
       const config = { tooltip: { _fn_formatter: '' } };
-      chart.__inflateFunctions(config);
+      inflateFunctions(config);
       expect(config.tooltip.formatter).to.be.a('function');
       expect(config.tooltip).to.not.have.property('_fn_formatter');
     });
+
+    it('should not try to inflate if a non-object value is passed', () => {
```

Comment:
It's not obvious how it is actually testing that, I guess by checking that there are no errors? Should we reflect that in the test name or in a comment?

Reply:
Refactored the test to spy on `Object.entries`. WDYT?

Reply:
Not so sure about it, since it relies on implementation details of the function, which can change. I don't have any better suggestions for improving the test procedure, apart from just adding a clarifying comment to your original version.

If we want to go with the spy, it should be reset afterwards since it is applied on a global.

Reply:
Yeah, you are right about the spy usage. Restored the previous version and added a comment about the test.

Also removed the fixture, since it's not needed for that portion of the tests.